src/components/Editor/Invitees/BbbAddRoomActions.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<NcActions v-if="isCreateBbbRoomButtonVisible"
+		:menu-title="$t('calendar', 'Add video link')"
+		force-title
+		force-menu
+		:disabled="isCreateBbbRoomButtonDisabled"
+		@open="getBbbRooms">
+		<template #icon>
+			<Video :size="20" />
+		</template>
+		<NcActionButton close-after-click @click="addBbbRoom(null)">
+			<template #icon>
+				<Plus :size="20" />
+			</template>
+			{{ $t('calendar', 'Create new room') }}
+		</NcActionButton>
+		<NcActionSeparator v-if="bbbRoomsList.length > 0" />
+		<NcActionButton v-for="room in bbbRoomsList"
```

Comment:
**issue:** This won't be very nice when you have like 10+ rooms. There could be some kind of limitation of the list, then some kind of « List more rooms » option that opens maybe a modal to help filtering and finding the correct room.

Otherwise, just use a multi-select?